docs/configuration_file.md

```diff
@@ -27,6 +27,8 @@ Some options are also mandatory.
 
 * `verbose` (*optional*, `boolean`) — enable verbose logging
 
+* `openshift_required_version` (*optional*, `string`) — required version to run against (adjusts build template as appropriate)
```

Comment:
Not exactly happy but couldn't really think of anything better. `openshift_expected_min_version`? `openshift_compat_version`?